src/rules/allDefinedPropertyMustBeDecored/allDefinedPropertyMustBeDecored.ts

```diff
@@ -0,0 +1,61 @@
+import {AST_NODE_TYPES, TSESLint} from "@typescript-eslint/utils";
+import {PropertyDefinition} from "@typescript-eslint/types/dist/generated/ast-spec";
+import {createRule} from "../../utils/createRule";
+import {typeCheckingDecorator} from "../../utils/classValidatorDecorators";
+
+const rule = createRule({
+    name: "all-defined-property-must-be-decored",
```

Comment:
missing pluralisation convention that's on all other rules for "property"
spelling issue on "decored"